package.json

```diff
@@ -24,6 +24,7 @@
     "mocha": "^3.4.2",
     "moment": "^2.0.0",
     "node-sass": "^4.0.0",
+    "prop-types": "^15.5.10",
```

Comment:
Okay, so rebase with master please so tests on CI will automatically ran and you have my green light ;)